src/index.ts

```diff
@@ -27,7 +27,8 @@ function loader(this: interfaces.Webpack, contents: string) {
     }
 
     const rawFilePath = path.normalize(this.resourcePath);
-    const filePath = utils.appendTsSuffixIfMatch(options.appendTsSuffixTo, rawFilePath);
+    let filePath = utils.appendTsSuffixIfMatch(options.appendTsSuffixTo, rawFilePath);
```

Comment:
The only different is when filename is matched by both options.appendTsxSuffixTo and options.appendTsSuffixTo.
In that case, if both part use rawFilePath then `.tsx` append will overwrite `.ts`, but i think it should become `.ts.tsx` so user may know that their regexes has collision.